orcestra/flightplan.py

```diff
@@ -389,3 +389,60 @@ def path_quickplot(path, sel_time, crossection=True):
             ax2.axvline(point, color="k", lw=1)
 
     return fig
+
+def calc_zonal_mean(field, lon_min, lon_max, lat_min, lat_max): 
+    import easygems.healpix as egh
+    
+    bbox = [lon_min, lon_max, lat_min, lat_max]
+    
+    if "lon"  in field.coords:
+        field = field.rename({'lon': 'longitude'})
+    if "lat"  in field.coords:
+        field = field.rename({'lat': 'latitude'})
```

Comment:
This probably isn't compatible with the latest version of easygems, which uses `lat` and `lon` for the coordinates by default.

Reply:
I am a bit unclear on how to resolve this one... One option would be, I guess, to fix the minimum version of easygems you need in the .toml file and then use <lat> and <lon> but I am not sure if this is what we really want?

Reply:
Yes, I guess a minimum version of easygems would be the best option.